alyssa_mckee/flask_fundamentals/Registration_Form/server.py

```diff
@@ -0,0 +1,56 @@
+from flask import Flask, render_template, redirect, request, session, flash
+import re 
+
+EMAIL_REGEX = re.compile(r'^[a-zA-Z0-9.+_-]+@[a-zA-Z0-9._-]+\.[a-zA-Z]+$')
+app = Flask(__name__)
+
+app.secret_key = "Z%6QD3boMxk2M9xQ@oiq&5fRJB!A4A8*l#@!!y%4bX"
+
+@app.route("/")
+def index():
+	return render_template("index.html")
+
+@app.route('/process', methods=["POST"])
+def process():
+	error = False
+	
```

Comment:
Great job with the validations. I would recommend trying to move your validation logic to a separate method. This is to make our functions well defined, and also if you want to validate your data inside another method, then it would also make that much easier.